Sources/Hub/HubApi.swift

```diff
@@ -201,6 +231,12 @@ public extension HubApi {
             repoDestination.appending(path: relativeFilename)
         }
         
+        var metadataDestination: URL {
+            repoDestination
+                .appendingPathComponent(".cache")
+                .appendingPathComponent("huggingface")
```

Comment:
```suggestion
                .appendingPathComponent("huggingface")
                .appendingPathComponent("download")
```

Maybe we should consider adding this, for parity with the hub cli